test/evp_extra_test.c

```diff
@@ -5935,6 +5935,7 @@ static int test_evp_cipher_pipeline(void)
     unsigned char *ciphertext, *exp_plaintext, *tag;
     size_t numpipes, plaintextlen, i;
 
+    EVP_CIPHER_free(EVP_CIPHER_fetch(testctx, "AES-256-GCM", NULL));
```

Comment:
Probably a better fix would be to use `"provider!=fake-pipeline"` instead of `testpropq` at line 5942.

Reply:
Ok, that works as well, with all 3 reproducers from #26276